### DISCUSSION TITLE
Ability to only allow worldedit in claims

### DISCUSSION THREAD

**Discussion by Jadiction** [GriefPrevention/GriefPrevention#747] [Ideas]
I know there a few plugins that used to provide something like this; however, they are outdated and the one I was able to load doesn't work. Essentially I'd mostly love to see players given the ability to `//wand`, `//set`, `//undo`, and `//redo` only within their claims. 

For creative world claims, this would surely be a highly used feature as WorldEdit is essential when building in creative mode to those who know how to use it. A feature within GriefPrevention or even an updated extension to provide this would be awesome.

**Comment by RoboMWM** [comment]
FAWE, amongst others, provide this functionality unless they've refactored it out

**Comment by Jadiction** [comment]
I'll try out FAWE, thanks.

**Comment by Jadiction** [comment]
FAWE's implementation of regional restriction works great. Unfortunately, their updates are slow and FAWE is in a bad state for 1.15.x as it's quite buggy and unstable. Hopefully, they can fix it up soon.
